cookbooks/swift/files/default/etc/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,35 @@
+scrape_configs:
+  - job_name: 'saio'
+    scrape_interval: 10s
+    static_configs:
+      - targets:
+        - 'localhost:9100'
+        labels:
+          host: 'proxy'
+
+      # backend servers
+      - targets:
+        - 'localhost:9101'
+        labels:
+          host: 'node1'
```

Comment:
This extra labelling is maybe not *strict* necessary (you'd be able to pick up on it from `instance` labels, which would look like `localhost:9101`), but

- I like the explicitness, and
- it shows how you can add even more context in the scrape config; things like rack or DC information